src/main/java/frc/robot/subsystems/internals/InternalSubsystem.java

```diff
@@ -99,24 +100,30 @@ public InternalSubsystem(TurretSubsystem turretSubsystem) {
             case Blue: ALLIANCE_COLOR = BLUE; break;
             default: ALLIANCE_COLOR = RED; break;
         }
+
+        // if we are in auton we start with 1 ball
+        if (DriverStation.isAutonomous()) {
+            totalBallCount = 1;
```

Comment:
If the "between two sensor" ball count ints are still used (they are still being set in periodic), this should set one of those ints to 1 instead of the total ball count, with total ball count just being a sum of the 3. If they're not used, probably deleting all 3 from periodic and the subsystem would be fine; just have `getBallCount()` return `totalBallCount`.